dll/cpl/desk/background.c

```diff
@@ -652,6 +653,7 @@ OnBrowseButton(HWND hwndDlg, PBACKGROUND_DATA pData)
 
     hwndBackgroundList = GetDlgItem(hwndDlg, IDC_BACKGROUND_LIST);
     himl = ListView_GetImageList(hwndBackgroundList, LVSIL_SMALL);
+    SHGetFolderPathW(NULL, CSIDL_MYPICTURES, NULL, 0, initialDir);
```

Comment:
What if SHGetFolderPathW fails ?

Reply:
Would this be acceptable? I found that desk uses this exact method a little further down in the code.
if (FAILED(SHGetFolderPath(NULL, CSIDL_MYPICTURES, NULL, 0, initialDir)))
    {
        return;
    }